integration-tests/docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
+  postgres:
+    image: postgres:latest
+    container_name: protect-ffi-postgres
+    environment:
+      POSTGRES_DB: cipherstash
+      POSTGRES_USER: cipherstash
+      POSTGRES_PASSWORD: password
```

I recommend you make this look up an environment variable first, then fall back to hard coded credentials:

```suggestion
      - POSTGRES_DB=${PGDATABASE:-cipherstash}
      - POSTGRES_USER=${PGUSER:-cipherstash}
      - POSTGRES_PASSWORD: =${PGPASSWORD:-password}
```

This means that:

- If you change the environment variables in the `test.yml` GitHub Actions workflow, those new credentials are automatically picked up
- Devs running this for the first time don't have to worry about setting environment variables